browser/resources/settings/brave_leo_assistant_page/brave_leo_assistant_page.ts

```diff
@@ -39,30 +39,27 @@ class BraveLeoAssistantPageElement extends BraveLeoAssistantPageBase {
     }
 
     leoAssistantShowOnToolbarPref_: boolean
-    leoAssistantShowPromptsPref_: boolean
     browserProxy_: BraveLeoAssistantBrowserProxy =
       BraveLeoAssistantBrowserProxyImpl.getInstance()
 
-    onResetAssistanceData_() {
-      this.browserProxy_.reset().then((result) => {
-        if(result) {
-          window.alert(this.i18n('braveLeoAssistantResetConfirmed'))
-          this.getShowLeoAssistantIcon_()
-        }
+    onResetAssistantData_() {
+      this.browserProxy_.resetLeoData().then((result) => {
+        if(!result)
```

Comment:
I think we can entirely ignore the promise result - it's always true (I take it this is because we have to return something?).

Reply:
Changed the return value for `resetLeoData` to `void`, as we don't need to return anything for now